Doc/library/argparse.rst

```diff
@@ -647,6 +653,28 @@ the help options::
      +h, ++help  show this help message and exit
 
 
+exit_on_error
+^^^^^^^^^^^^^
+
+Normally, when you pass an wrong argument list to the :meth:`~ArgumentParser.parse_args`
```

Comment:
A basic grammar correction would be to just change "an" to "a", but I think that "invalid" makes more sense in this context than "wrong":

```suggestion
Normally, when you pass an invalid argument list to the :meth:`~ArgumentParser.parse_args`
```